### ISSUE TITLE
Generated documentation in bindings.rs improperly formatted

### ISSUE THREAD

**Issue by noxabellus** [rustwasm/wasm-bindgen#1889]
### Describe the Bug

The generated bindings.rs file for web-sys has inline documentation that does not appear to be formatted properly. In VS Code it does not always get displayed correctly in tooltips.

### Steps to Reproduce

1. Open web-sys example in VS Code
2. Hover over items in main (e.g. window.document())
3. Contextual info is displayed improperly

### Expected Behavior

![image](https://user-images.githubusercontent.com/13547335/70015390-d1329780-1542-11ea-85be-316a52b3af11.png)

### Actual Behavior

![image](https://user-images.githubusercontent.com/13547335/70015408-dc85c300-1542-11ea-869e-da28b06f6c8a.png)

**Comment by noxabellus** [comment]
It seems there is a combination of issues here. The first of which is the minified style of the output. Running rustfmt on the bindings.rs file cleans up the display so that you don't see raw code, but VS Code does not show past the first line of the documentation.

After running rustfmt VS Code shows this:
![image](https://user-images.githubusercontent.com/13547335/70016082-f922fa80-1544-11ea-8ed8-01fc317ac973.png)

However, the full doc line for `document` is this:
```rust
#[doc = "The `document` getter\n\n[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/Window/document)\n\n*This API requires the following crate features to be activated: `Document`, `Window`*"]
```
This second problem of not showing the full doc seems to be a standing issue with RLS+VS Code, but its worth noting that functions that are documented with the triple-slash comment style `///` are displayed correctly.

E.g. the `window` function show in my first comment is documented this way:
```rust
/// Getter for the `Window` object
///
/// [MDN Documentation]
///
/// *This API requires the following crate features to be activated: `Window`*
///
/// [MDN Documentation]: https://developer.mozilla.org/en-US/docs/Web/API/Window
#[cfg(feature = "Window")]
pub fn window() -> Option<Window> { ...
```

**Comment by alexcrichton** [comment]
Thanks for the report! The fix for this in the past is to [run `rustfmt` over the generated bindings](https://github.com/rustwasm/wasm-bindgen/blob/df34cf843eca7478e3879562670e52c889e32fdf/crates/web-sys/build.rs#L92-L98), which should happen by default. Do you know where RLS is getting its documentation from? Is it running somewhere without `rustfmt` available?

**Comment by noxabellus** [comment]
When rls runs on a project it creates an rls directory in the project's build directory and does a pretty standard build from what I can tell. When I use go to definition, it opens `target\rls\debug\build\web-sys-9e28686fc4046c2a\out\bindings.rs`, so I assume this is the same location it gets the documentation from.

After I installed rustfmt with `rustup component add rustfmt` it does indeed automatically run rustfmt over the bindings.

As I said though, this only solves half the issue as rls doesn't fully/properly support `#[doc]` annotations; it seems to expect triple-slash `///` style doc comments instead. To test that theory I created a nodejs script that replaces all the `#[doc]` with `///` (and converts the escaped newlines to actual newlines) and they are indeed displayed fully/properly afterwards.
